benchmark/launch_software.sh

```diff
@@ -77,6 +77,8 @@ gi_to_taxid_fp=${26}
 parse_hgts=${27}
 # EGID install directory
 egid_install_dp=${28}
+# GeneMark install directory
+genemark_install_dp=${29}
 # qsub params
 qsub="-q route -m abe -M jenya.kopylov@gmail.com -l nodes=1:ppn=${threads} -l walltime=230:00:00 -l pmem=10gb -l mem=20gb"
```

Comment:
Moreover, because there are so many arguments (>30), how about putting them into a configuration file, instead of command-line arguments?